### ISSUE TITLE
way of tode doesn't follow way of tode

### ISSUE THREAD

**Issue by TodePond** [l2wilson94/WayOfTode#2]
something doesn't feel write about the way of tode so far. It's too wordy and complicated + detailed. It doesn't follow my mantra of keeping things brief + simple at all. I'm gonna play around with some different formats in the `wip` folder.

**Comment by TodePond** [comment]
I tried a different format but still no good. I think the way of tode would be better off as a multi-page thing. I've started drafting up some things at https://l2wilson94.gitbook.io/way-of-tode/

**Comment by Magnogen** [comment]
Wow that really is simple
